### PR TITLE
upgrade solvers to latest version and decommision solver-f32-py37

### PR DESCRIPTION
**Pull request by harshad16**
upgrade solvers to latest version and decommision solver-f32-py37
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

## Related Issues and Dependencies

Related-to: #809

### CODE REVIEW

File: core/overlays/ocp4-stage/common/configmaps.yaml [hunk 1]
```diff
@@ -38,7 +38,6 @@ data:
   solvers: |
     solver-rhel-8-py36
     solver-rhel-8-py38
-    solver-fedora-32-py37
```
Comment (goern) [inline core/overlays/ocp4-stage/common/configmaps.yaml:41]: it does not be decommissioned in cnv-prod?

**Review by fridex — APPROVED** [review]
thanks for this change :ok_hand:

File: core/overlays/ocp4-stage/common/configmaps.yaml (hunk 1 above)
Comment (harshad16) [reply core/overlays/ocp4-stage/common/configmaps.yaml:41]: we dont run fedora solver in prod yet .

Comment (fridex) [reply core/overlays/ocp4-stage/common/configmaps.yaml:41]: We wanted to sync with s2i provided. We do not have f32-py37 https://github.com/thoth-station/s2i-thoth/

**Pull request merged**